deploy-agent/deployd/staging/transformer.py

```diff
@@ -73,7 +73,7 @@ def _translate(self, from_path, to_path):
             log.error('Fail to translate script {}, stacktrace: {}'.format(from_path,
                                                                            traceback.format_exc()))
 
-    def transform_scripts(self, script_dir, template_dirname, script_dirname):
+    def transform_scripts(self, script_dir, template_dirname, script_dirname) -> list:
```

Comment:
probably we should be consistent to use either list or List from typing. wdyt?

Reply:
good catch, thanks!